hydra-redis/querying_mechanism.py

```diff
@@ -0,0 +1,184 @@
+import redis
+import hydra_graph as graph
+from hydrus.hydraspec import doc_maker
+import hydrus
+from collections_endpoint import CollectionEndpoints
+import urllib.request
+import json
+
+def final_file(url):
+    """ Open the given url and read and load the Json data."""
+    response = urllib.request.urlopen(url)
```

Comment:
`try-except` block needed here. Every I/O call has to be safely written into an exception block, same for the JSON serialization below.